.github/workflows/cacti-dev-container-vscode-publish.yaml

```diff
@@ -1,7 +1,7 @@
 name: connector-fabric-publish
 
 env:
-  NODEJS_VERSION: v20.3.0
+  NODEJS_VERSION: v20.11.1
```

Comment:
@ashnashahgrover NodeJS v18 is still being maintained (will be EOL'd next Summer) for about 12 months and I know @outSH prefers to keep the baseline at v18 for that reason (so that while v18 is still being maintained we support it as well which is easiest to ensure by using it ourselves)

Long story short, please only do minor or patch upgrades on the Node version for now.